### DISCUSSION TITLE
Unable to verify my academic status

### DISCUSSION THREAD

**Discussion by lostl1ght** [community/community#56620] [GitHub Education]
### Hi everyone,

I have uploaded every document I have that can theoretically verify my eligibility for Github Student Pack but I still cannot get access to it. Please help.

**Comment by Ntshangase** [comment]
Hi @lostl1ght 

It looks like your student benefits have been approved, you can confirm here: [Click Me](https://education.github.com/discount_requests/application)

**Comment by samus-aran** [comment]
Hi @lostl1ght 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
